src/proxy-ui-api/build.gradle

```diff
@@ -208,6 +222,7 @@ dependencies {
     implementation('wsdl4j:wsdl4j:1.6.3')
     implementation('com.github.vladimir-bukhtoyarov:bucket4j-core:4.10.0')
     implementation("io.swagger.parser.v3:swagger-parser:${swaggerParserVersion}")
+    implementation("org.postgresql:postgresql:42.2.14")
```

Comment:
should be unnecessary (already a transitive dependency)

Reply:
Fixed